values-schema.yaml

```diff
@@ -2114,7 +2120,7 @@ properties:
           apiServer:
             additionalProperties: false
             properties:
-              enabled:
+              create:
```

Comment:
is create the right property?

Reply:
It was the one currently used in the defaults. So it was either change the value in the defaults.yaml to enabled or change the openapi schema.

Reply:
I check it is a correct change. I also added the values migration for the schema change